lava-master/scripts/setup.sh

```diff
@@ -56,9 +61,14 @@ if [ -e /root/device-types ];then
 	do
 		cp $i /etc/lava-server/dispatcher-config/device-types/
 		devicetype=$(basename $i |sed 's,.jinja2,,')
-		echo "Adding custom $devicetype"
-		lava-server manage device-types add $devicetype || exit $?
-		touch /root/.lavadocker/devicetype-$devicetype
+		lava-server manage device-types list | grep -q "[[:space:]]$devicetype[[:space:]]"
+		if [ $? -eq 0 ];then
+			echo "Skip $devicetype"
```

Comment:
Similarily, not a helpful message, and this change is not described in the changelog.